overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -88025,7 +88025,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "Unstable to Lunar",
-          "desc:8": "Craft some Unstable Ingots and upgrade them immediately into some Moon Stones before they blow up! Make sure to have every material in your inventory, and do the second step of the crafting under 10 seconds, before the Unstable Ingots gets you.\n\nOr alternatively if you\u0027re rolling in Diamonds, you can spend 9 times the Diamond to craft Stable-\u0027Unstable\u0027 Nuggets and put them together into a Stable-\u0027Unstable\u0027 Ingot, which doesn\u0027t blow up, but can be used for crafting the same way."
+          "desc:8": "Craft some Unstable Ingots and upgrade them immediately into some Moon Stones before they blow up! Make sure to have every material in your inventory, and do the second step of the crafting under 10 seconds, before the Unstable Ingots gets you.\n\nOr alternatively if you\u0027re rolling in Diamonds, you can spend 9 times the Diamond to craft Stable-\u0027Unstable\u0027 Nuggets and put them together into a Stable-\u0027Unstable\u0027 Ingot, which doesn\u0027t blow up, but can be used for crafting the same way.\n\nNote: Contrary to what the tooltip on the Unstable Ingot says, you can craft it in any crafting container. However, it still blows up after the timer expires. Even if you later put them into an ME System, if you take them out after their timer has reached zero, you die."
```

Comment:
with #1003 the offending tooltip has been removed, and so this statement should be corrected - the change here is probably not even needed anymore.

Reply:
To be honest, I would leave it in. Or add a tooltip to the Unstable Ingot telling the player that it can be crafted in any crafting container. Just because I think that people are so used to only being able to craft the Unstable Ingot in a vanilla crafting table, that the removed tooltip will probably not be enough to let players know that it doesn't have to be a vanilla workbench. 
If you feel like this should definitely go (not "probably" as you wrote in your review), I can remove it, I just wanted to share my opinion.